laser_slam_ros/include/laser_slam_ros/laser_slam_worker.hpp

```diff
@@ -43,6 +43,10 @@ class LaserSlamWorker {
   // Get a filtered map and apply map separation if desired.
   void getFilteredMap(laser_slam_ros::PointCloud* filtered_map);
 
+  // Get a vector containing the point clouds that have been recorded since
```

Comment:
I would try to rephrase recorded to indicate that the node associated to the scans which are queued have been optimized by the back-end.